index.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.4.0/css/font-awesome.min.css">
+  <link href="style.css" rel="stylesheet" type="text/css">
+  <title>Homepage of YHS</title>
+</head>
+
+<body>
+
+  <header class="container">
+    <nav class="nav navbar-one">
+      <a class="nav-item" href="#"><i class="fa fa-facebook fa_custom"></i></a>
+      <a class="nav-item" href="#"><i class="fa fa-twitter"></i></a>
+      <a class="nav-item" href="#">Logout</a>
+      <a class="nav-item" href="index.html">Homepage</a>
+      <a class="nav-item" href="#">English</a>
+    </nav>
+    <nav class="navbar-two">
+      <div class="navbar-header">
+        <div class="humburger">
+          <span class="icon-bar"></span>
+          <span class="icon-bar"></span>
+          <span class="icon-bar"></span>
+        </div>
+        <a id="header-logo" href="index.html"><img src="./img/NewLogo4.png" alt="Logo"></a>
+      </div>
+      <div class="collapse navbar-collapse" id="myNavbar">
+        <a class="link" href="index.html">Home</a>
+        <a class="link" href="about.html">About</a>
+        <a class="link" href="#">Program</a>
+        <a class="link" href="#">Join</a>
+        <a class="link" href="#">Sponsors</a>
+        <a href="#" id="campaign" class="link">YHS Campaign</a>
+      </div>
+    </nav>
+  </header>
+
+  <!--#################### MAIN SECTION ###################-->
+  <main class="main-container">
+    <section class="landing-page-section">
+      <p class="greetings">"Hello! Welcome to YHS"</p>
+      <h1 class="landing-page-header">YOUTH HOPE SUMMIT <br>2022</h1>
+      <p class="landing-page-para">
+        A joyful celebration believing in the value of opennes and sharing, creating a positive change with people from
+        over 80 countries is taking place in August, in Ghana.
+      </p>
+      <h2 class="duration">2022.08.19 (FRI) ~ 20(SAT)</h2>
+      <p class="location">@ National Theater, Accra-Ghana</p>
+    </section>
```

Comment:
- [ ] Kindly follow spacing. border, margins here as shown in the attached loom video as per the project requirements.